.github/workflows/codeql.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branch: [ 'master', 'branch/v11', 'branch/v12', 'branch/v13' ]
+        branch: [ 'master', 'branch/v12', 'branch/v13', 'branch/v14' ]
```

Comment:
Should we keep branch/v11 for now?

Reply:
For CodeQL I think it is safe to drop v11.

We're getting little negative signal out of these the recurring checks AFAIK.  The hassle of remembering to remove v11 is a greater cost to the org than the chance we miss something important on v11 in the next week.

Reply:
@wadells is echo'ing my thoughts as well.  It's a small time window and any changes in v11 should go through other branches first anyways.  So the window of risk is small and since this is a workflow update this PR needs significant approval: https://github.com/gravitational/teleport/blob/master/.github/CODEOWNERS#L3